cmd/agent/main.go

```diff
@@ -146,6 +147,7 @@ func mergeOrError(path string) {
 	v.SetDefault("orb.db.file", "./orb-agent.db")
 	v.SetDefault("orb.tls.verify", true)
 	v.SetDefault("orb.otel.enable", false)
+	v.SetDefault("orb.otel.type", "prometheus")
```

Comment:
"type" is too generic, please make this "receiver_type" or similar. we do plan to deprecate this essentially immediately, right?

Reply:
done. and you are right